x-pack/legacy/plugins/security/public/views/login/components/basic_login_form/__snapshots__/basic_login_form.test.tsx.snap

```diff
@@ -64,6 +64,7 @@ exports[`BasicLoginForm renders as expected 1`] = `
       >
         <EuiFieldText
           aria-required={true}
+          autoComplete="off"
```

Comment:
```suggestion
          autocomplete="off"
```
Supernit, ["autocomplete" is entirely lowercase](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

Reply:
Request uses a non-standard capitalization of `autoComplete`.